utility/src/main/java/org/oppia/android/util/logging/BUILD.bazel

```diff
@@ -149,19 +149,6 @@ kt_android_library(
     ],
 )
 
-kt_android_library(
-    name = "kenya_alpha_event_logging_configuration_module",
-    srcs = [
-        "KenyaAlphaEventLoggingConfigurationModule.kt",
-        "KenyaAlphaEventTypeToHumanReadableNameConverterImpl.kt",
```

Comment:
Can you also clean up this bit? We no longer need to have a separate interface/implementation for the event type to readable name converter. Specifically:

https://github.com/oppia/oppia-android/blob/develop/utility/src/main/java/org/oppia/android/util/logging/StandardEventTypeToHumanReadableNameConverterImpl.kt

Can be inlined into:

https://github.com/oppia/oppia-android/blob/develop/utility/src/main/java/org/oppia/android/util/logging/EventTypeToHumanReadableNameConverter.kt

I think this will allow removing EventLoggingConfigurationModule as well.

Reply:
I searched the usage for EventLoggingConfigurationModule which has 177 usages in tests and components. I just want to get an confirmation and I can remove them all. Hope removing them all won't cause any problem.

Reply:
Additional kenyaAlpha clean done, Inlined to [EventTypeToHumanReadableNameConverter.kt], and clean the EventLoggingConfigurationModule.

Reply:
Ah sorry--missed the earlier comment re: confirmation. I can confirm the approach looks correct!